src/lib.rs

```diff
@@ -21,17 +21,19 @@
 //! guid.data3();
 //! guid.data4();
 //! ```
+#![forbid(unsafe_code)]
+#![deny(clippy::all)]
```

Comment:
I'd rather not add these.

Reply:
Im ok with removing them, but may I ask why?

Reply:
Both are restrictive.

Denying unsafe code would flag to any future contributor that unsafe code would _never_ be accepted, which, although unlikely, is not true.

Restricting clippy can also increase friction for contribution, and this crate is small enough that the gain from clippy does not really outweigh the cost. In saying that, I do think in general running clippy and implementing the suggestions are worth it.
For a larger project, I would probably alter the CI to deny clippy (and also rustfmt).

Reply:
Ok fair point. I just use them as a general default sometimes. I view them more as guiding rails. Questioning myself things like: "Does there really need to be unsafe code?" And the clippy one, I see it not as restricting clippy to only use that rule as it just overwrites the level of output, meaning it will not only warn but deny to compile.

Reply:
But got you, removed them